src/engine/enginebuffer.cpp

```diff
@@ -448,6 +458,7 @@ void EngineBuffer::readToCrossfadeBuffer(const int iBufferSize) {
 void EngineBuffer::setNewPlaypos(double newpos) {
     //qDebug() << m_group << "engine new pos " << newpos;
 
+    assertValidPlayPosition(newpos);
```

Comment:
Are you sure? There is another check including a correction of the parameter 
value in EngineBuffer::doSeekPlayPos()!

What's the difference between setNewPlaypos() and doSeekPlayPos()?

On 27.05.2016 23:47, Daniel Schürmann wrote:
>
> In src/engine/enginebuffer.cpp 
> <https://github.com/mixxxdj/mixxx/pull/954#discussion_r64970724>:
>
> > @@ -448,6 +458,7 @@ void EngineBuffer::readToCrossfadeBuffer(const int iBufferSize) {
> >  void EngineBuffer::setNewPlaypos(double newpos) {
> >      //qDebug() << m_group << "engine new pos " << newpos;
> >
> > +    assertValidPlayPosition(newpos);
>
> It is fully valid to seek to a fractional frame.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/954/files/78c1ea84111198fd32ecdd84a6ed448fd103f145#r64970724>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABZ-atoYsxbJH8uMjtCzXWSocXZ_tsdyks5qF2aBgaJpZM4ImsNg>.
>